lib/phoenix_test/utils.ex

```diff
@@ -13,4 +15,28 @@ defmodule PhoenixTest.Utils do
         {to_string(k), to_string(v)}
     end)
   end
+
+  def current_endpoint() do
+    current_app = Mix.Project.config()[:app]
+
+    if is_nil(current_app) do
+      raise "no :app set in Mix project config"
```

Comment:
What do you think about having a nicer error message here? If a user gets this error without knowing that it's due to the config, it might not be clear _why_ they need to set the `:app` and for which library. Maybe we could suggest something here, or let them know what we expected? 

```suggestion
      raise "could not find `:my_app, :phoenix_test` set in project config. Please set it along with your endpoint."
```

I'm not sure if that's too verbose, but maybe something like that?